### ISSUE TITLE
first argument of VLOOKUP is not coerced from null

### ISSUE THREAD

**Issue by izulin** [handsontable/hyperformula#526]
### Description
If first argument of VLOOKUP is null, it should be coerced to 0.

```
it('should coerce empty arg to 0', () => {
  const engine = builder([
    ['0', 'a'],
    ['2', 'b'],
    ['3', 'c'],
    ['4', 'd'],
    ['5', 'e'],
    ['=VLOOKUP(C3, A1:B5, 2)'],
  ])
  expect(engine.getCellValue(adr('A6'))).toEqual('a')
})
```

### Steps to reproduce

1.
2.
3.

### Your environment

* HyperFormula version:
* Browser Name and version:
* Operating System:

**Comment by voodoo11** [comment]
Fixed in #520

**Comment by wojciechczerniak** [comment]
Awesome 🏆 Thanks
